app/components/blacklight/document/page_header_component.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Blacklight
+  module Document
+    # Render the start over and prev/next displays
+    class PageHeaderComponent < Blacklight::Component
+      attr_reader :document, :blacklight_config, :search_context, :search_session
+
+      delegate :blacklight_config, to: :helpers
+
+      def initialize(document:, search_context:, search_session:)
+        super
+        @search_context = search_context
+        @search_session = search_session
+        @document = document
+      end
+
+      def render?
+        search_context.present? || search_session.present? || has_header_tools?
+      end
+
+      def applied_params_component
+        return unless blacklight_config.track_search_session.applied_params_component
+
+        blacklight_config.track_search_session.applied_params_component.new
+      end
+
+      def pagination_component
+        return unless blacklight_config.track_search_session.item_pagination_component
+
+        blacklight_config.track_search_session.item_pagination_component.new(search_context: search_context, search_session: search_session, current_document: document)
```

Comment:
```suggestion
        blacklight_config.track_search_session.item_pagination_component.new(search_context:, search_session:, current_document: document)
```

Reply:
I did this originally. Rubocop complains.

Reply:
I see that in some `.erb` files you did this too.  So can we be consistent in the style we're using?